main.tf

```
@@ -51,6 +53,31 @@ data "aws_iam_policy_document" "default" {
   }
 }
 
+data "aws_iam_policy_document" "ssm_patch_s3_log_policy" {
+  count = local.ssm_path_log_bucket_enabled ? 1 : 0
+  statement {
+    sid = "AllowAccessToPathLogBucket"
+    actions = [
+      "s3:GetObject",
+      "s3:PutObject",
+      "s3:PutObjectAcl",
+      "s3:GetEncryptionConfiguration",
+    ]
+    resources = [
+      "arn:aws:s3:::${var.ssm_patch_manager_s3_log_bucket}/*",
+      "arn:aws:s3:::${var.ssm_patch_manager_s3_log_bucket}",
+    ]
+  }
+}
+
+resource "aws_iam_policy" "ssm_patch_s3_log_policy" {
+  count       = local.ssm_path_log_bucket_enabled ? 1 : 0
+  name        = "${module.this.enabled}-ssm-patch-s3-logs"
```

```suggestion
  name        = "${module.this.id}-ssm-patch-s3-logs"
```

our pattern is to use a separate `label` module in this case and add the attribute

```
module "label_ssm_patch_s3_log_policy" {
  source     = "cloudposse/label/null"
  version    = "0.24.1"
  
  attributes = ["ssm-patch-s3-logs"]
  context    = module.this.context
}

name        = module.label_ssm_patch_s3_log_policy.id
```